articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,55 @@
+### Introduction
+
+Technology is growing at a fast pace in recent years. Due to this, it is now easy to get lost in the rumpus of technological terms that keep popping up every day. Machine learning and deep learning are among the fast-growing technologies. Besides this two, [Internet of Things (IoT](https://internetofthingsagenda.techtarget.com/definition/Internet-of-Things-IoT)), [blockchain](https://en.wikipedia.org/wiki/Blockchain), and [Artificial Intelligence](https://www.investopedia.com/terms/a/artificial-intelligence-ai.asp) (AI) are among the fast-growing technologies.
+
```

Comment:
Change: Besides this two -> 'Besides these two'

Reply:
I have omitted it since I have changed my sentence structure.